### PR TITLE
fix(node_manager): lowercase owner for discord

### PR DESCRIPTION
**Pull request by mazzi**
### Description

We use owner in lowercase to be able to compute scores accordingly on Discord.

Related https://github.com/maidsafe/dags-service/pull/7
Related https://github.com/maidsafe/safe_network/pull/2180

### CODE REVIEW

File: sn_node_manager/src/add_services/mod.rs [hunk 1]
```diff
@@ -78,6 +78,11 @@ pub async fn add_node(
         check_port_availability(port_option, &node_registry.nodes)?;
     }
 
+    let owner = match &options.owner {
+        Some(owner) => Some(owner.to_lowercase()),
```
Comment (maqi) [inline sn_node_manager/src/add_services/mod.rs]: shall we raise a warning here in case an `UpperCase` input detected ?

at least a `warning log` ?

**Review by maqi — APPROVED** [review]

File: sn_node_manager/src/add_services/mod.rs (hunk 1 above)
Comment (mazzi) [reply sn_node_manager/src/add_services/mod.rs]: Good idea.

**Comment by jacderida** [comment]
Hey mazzi,

The node manager has a test suite. It would be nice if we could add a test case for this. The tests can be a bit unwieldy though because they use a lot of mocking and stuff, so if it's too much hassle, don't worry about it.

**Comment by mazzi** [comment]
> Hey mazzi,
> 
> The node manager has a test suite. It would be nice if we could add a test case for this. The tests can be a bit unwieldy though because they use a lot of mocking and stuff, so if it's too much hassle, don't worry about it.

I've modified an existing test. I didn't wanted to copy past the one that I've modified.
Did you see it? It's alright?
Thanks!

**Comment by jacderida** [comment]
> > Hey mazzi,
> > The node manager has a test suite. It would be nice if we could add a test case for this. The tests can be a bit unwieldy though because they use a lot of mocking and stuff, so if it's too much hassle, don't worry about it.
> 
> I've modified an existing test. I didn't wanted to copy past the one that I've modified. Did you see it? It's alright? Thanks!

Right, thanks. However, wouldn't it need an assertion that the assigned value is in lowercase?

**Comment by mazzi** [comment]
> > > Hey mazzi,
> > > The node manager has a test suite. It would be nice if we could add a test case for this. The tests can be a bit unwieldy though because they use a lot of mocking and stuff, so if it's too much hassle, don't worry about it.
> > 
> > 
> > I've modified an existing test. I didn't wanted to copy past the one that I've modified. Did you see it? It's alright? Thanks!
> 
> Right, thanks. However, wouldn't it need an assertion that the assigned value is in lowercase?

The assertion was already there. I just modified the input. Please check the whole test.

**Review by jacderida — APPROVED** [review]

**Comment by jacderida** [comment]
> > > > Hey mazzi,
> > > > The node manager has a test suite. It would be nice if we could add a test case for this. The tests can be a bit unwieldy though because they use a lot of mocking and stuff, so if it's too much hassle, don't worry about it.
> > > 
> > > 
> > > I've modified an existing test. I didn't wanted to copy past the one that I've modified. Did you see it? It's alright? Thanks!
> > 
> > 
> > Right, thanks. However, wouldn't it need an assertion that the assigned value is in lowercase?
> 
> The assertion was already there. I just modified the input. Please check the whole test.

Ah ok cool, sorry. Approved.

**Pull request merged**